FusionAuth/fusionauth-javascript-sdk
# 46 Add postLogoutRedirectUri Param Handling

## What is this PR and why do we need it?

This adds the `postLogoutRedirectUri` param into the SDK config and updates logic for SDK to redirect users to the optionally defined `postLogoutRedirectUri`, if this does not exist the user should be brought to the `redirectUri` by default.

To Test:

1. Run unit tests to ensure updated urlHelper Tests pass when postLogoutRedirectUri param is passed in
2. Define a postLogoutRedirectUri in the config of the consuming application and ensure the url is accepted by adding it to the application configuration in the admin dashboard.
3. Log In to the consuming application
4. Log Out and ensure that when postLogoutRedirectUri is defined that redirect to the defined param is successful, if it is not defined ensure that redirect to the redirectUri defaults

#### Pre-Merge Checklist (if applicable)
